ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.9__amd64-stable.yaml

```diff
@@ -341,15 +341,27 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-azure-ipi-private
-- as: azure-ipi-private-p2-f14
-  cron: 34 3 14,29 * *
+- as: azure-ipi-private-workers-marketimage-p2-f14
+  cron: 51 17 13,28 * *
   steps:
-    cluster_profile: azure-qe
+    cluster_profile: azure-marketplace-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
     test:
     - ref: cucushift-installer-check-cluster-health
-    workflow: cucushift-installer-rehearse-azure-ipi-private
+    workflow: cucushift-installer-rehearse-azure-ipi-private-workers-marketimage
+- as: azure-ipi-private-workers-marketimage-p3-f28-destructive
```

Comment:
Based on latest test strategy for stable payloads, we are making a change to replace e2e with healthcheck, so the normal job and destructive job actually only do install the cluster and cluster health check, which make them the same now. So the destructive job is no longer needed here.

Reply:
got it, removed.